main_server.py

```
@@ -0,0 +1,445 @@
+                        else:
+                            analysis_data["text_input"] = f"Content from {file.filename}:\n{file_content}"
+                    except Exception as e:
+                        file_info["error"] = f"Text file processing error: {str(e)}"
+                
+                analysis_data["files_processed"].append(file_info)
+                
+                # Clean up temporary file
+                try:
+                    os.unlink(tmp_file_path)
+                except:
+                    pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)